integrationtests/tests/tests.rs

```diff
@@ -1199,3 +1164,38 @@ async fn verifies_client_configs() -> Result<()> {
     })
     .await
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn cannot_replay_transactions() -> Result<()> {
+    test(4, |fed, user, bitcoin, _, _| async move {
+        fed.mine_and_mint(&user, &*bitcoin, sats(5000)).await;
```

Comment:
Need to call `fed.prepare_funding_wallet` to ensure funds are available.

Not sure if we should just do it from `test`, because 95% of tests will want it...

Reply:
Yeah, let's do that as a follow-up refactor, for now I'll add it to the test.